app/helpers/guides_helper.rb

```diff
@@ -2,4 +2,24 @@ module GuidesHelper
   def book_an_appointment_link?
     !@guide.related_to_appointments?
   end
+
+  def previous_guide
+    return @previous_guide if defined?(@previous_guide)
```

Comment:
That would cause the code to be run twice for the nil option, while this approach avoids that duplication.